---
result: FAILURE
url: http://wilson.ci.chef.co/job/chef-server-12-trigger-ad_hoc/12/
failure_category: unknown
failure_cause: 'unknown: el, ubuntu-14.04; chef-client: ubuntu-10.04, ubuntu-12.04:
  chef-server-12-test'
timestamp: 2015-12-29 12:58:17 UTC
duration: 1h43m29s
triggered_by: jaym
active_duration: 4h28m43s
parameters:
  GIT_REF: jdm/fips-final
  EXPIRE_CACHE: false
change:
  git_remote: git@github.com:chef/chef-server.git
  git_commit: c8de0f4dedd2983be792939d1e08af6c6088df3d
  project: chef-server
  version: 12.3.1+20151229125832
stages:
  chef-server-12-build/architecture=x86_64,platform=el-6,project=chef-server,role=builder:
    result: SUCCESS
    url: http://wilson.ci.chef.co/job/chef-server-12-build/architecture=x86_64,platform=el-6,project=chef-server,role=builder/22/
    duration: 1h30m57s
  chef-server-12-build/architecture=x86_64,platform=el-5,project=chef-server,role=builder:
    result: SUCCESS
    url: http://wilson.ci.chef.co/job/chef-server-12-build/architecture=x86_64,platform=el-5,project=chef-server,role=builder/22/
    duration: 1h14m32s
  chef-server-12-test:
    result: FAILURE
    failure_category: unknown
    failure_cause: 'unknown: el, ubuntu-14.04; chef-client: ubuntu-10.04, ubuntu-12.04'
    url: http://wilson.ci.chef.co/job/chef-server-12-test/21/
    duration: 11m8s
    runs:
      el-5:
        result: FAILURE
        failure_category: unknown
        failure_cause: unknown
        url: http://wilson.ci.chef.co/job/chef-server-12-test/architecture=x86_64,platform=el-5,project=chef-server,role=tester/21/
        duration: 11m8s
        logs:
          consoleText:
            2530: |
              No entry for terminal type "unknown";
              using dumb terminal settings.
              /opt/opscode/embedded/service/oc-chef-pedant/lib/pedant/platform.rb:336:in `block in create_org': Bad error code 500 from create org: {"error":["internal service error"]} (RuntimeError)
                from /opt/opscode/embedded/service/oc-chef-pedant/lib/pedant/platform.rb:321:in `times'
                from /opt/opscode/embedded/service/oc-chef-pedant/lib/pedant/platform.rb:321:in `create_org'
            2543: |
              Configured URL: https://ip-10-194-15-112.us-west-2.compute.internal
              Creating org pedant_testorg_ip-10-194-15-112_17961
              Build step 'Invoke XShell command' marked build as failure
              Performing Post build task...
              Match found for : : True
            2564: |
              Recording fingerprints
              Recording test results
              ERROR: Publisher hudson.tasks.junit.JUnitResultArchiver aborted due to exception
              hudson.AbortException: No test report files were found. Configuration error?
                at hudson.tasks.junit.JUnitParser$ParseResultCallable.invoke(JUnitParser.java:116)
                at hudson.tasks.junit.JUnitParser$ParseResultCallable.invoke(JUnitParser.java:92)
                at hudson.FilePath$FileCallableWrapper.call(FilePath.java:2688)
                at hudson.remoting.UserRequest.perform(UserRequest.java:121)
                at hudson.remoting.UserRequest.perform(UserRequest.java:49)
                at hudson.remoting.Request$2.run(Request.java:325)
                at hudson.remoting.InterceptingExecutorService$1.call(InterceptingExecutorService.java:68)
                at java.util.concurrent.FutureTask.run(FutureTask.java:262)
                at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
                at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
                at java.lang.Thread.run(Thread.java:745)
                at ......remote call to chef-server-rhel-5-tester-4e6602(Native Method)
                at hudson.remoting.Channel.attachCallSiteStackTrace(Channel.java:1360)
                at hudson.remoting.UserResponse.retrieve(UserRequest.java:221)
                at hudson.remoting.Channel.call(Channel.java:753)
                at hudson.FilePath.act(FilePath.java:978)
                at hudson.FilePath.act(FilePath.java:967)
                at hudson.tasks.junit.JUnitParser.parseResult(JUnitParser.java:89)
                at hudson.tasks.junit.JUnitResultArchiver.parse(JUnitResultArchiver.java:121)
                at hudson.tasks.junit.JUnitResultArchiver.perform(JUnitResultArchiver.java:138)
                at hudson.tasks.BuildStepCompatibilityLayer.perform(BuildStepCompatibilityLayer.java:74)
                at hudson.tasks.BuildStepMonitor$1.perform(BuildStepMonitor.java:20)
                at hudson.model.AbstractBuild$AbstractBuildExecution.perform(AbstractBuild.java:761)
                at hudson.model.AbstractBuild$AbstractBuildExecution.performAllBuildSteps(AbstractBuild.java:721)
                at hudson.model.Build$BuildExecution.post2(Build.java:183)
                at hudson.model.AbstractBuild$AbstractBuildExecution.post(AbstractBuild.java:670)
                at hudson.model.Run.execute(Run.java:1766)
                at hudson.matrix.MatrixRun.run(MatrixRun.java:146)
                at hudson.model.ResourceController.execute(ResourceController.java:98)
                at hudson.model.Executor.run(Executor.java:374)
              [BFA] Scanning build for known causes...
              [BFA] No failure causes found
        steps:
          total: 11m8s
          before (setup time): 3m52s
          chef-client private-chef::default: 5m10s
          after (cleanup time): 2m5s
      el-6:
        result: FAILURE
        failure_category: unknown
        failure_cause: unknown
        url: http://wilson.ci.chef.co/job/chef-server-12-test/architecture=x86_64,platform=el-6,project=chef-server,role=tester/21/
        duration: 8m18s
        logs:
          consoleText:
            2506: |
              No entry for terminal type "unknown";
              using dumb terminal settings.
              /opt/opscode/embedded/service/oc-chef-pedant/lib/pedant/platform.rb:336:in `block in create_org': Bad error code 500 from create org: {"error":["internal service error"]} (RuntimeError)
                from /opt/opscode/embedded/service/oc-chef-pedant/lib/pedant/platform.rb:321:in `times'
                from /opt/opscode/embedded/service/oc-chef-pedant/lib/pedant/platform.rb:321:in `create_org'
            2519: |
              Configured URL: https://ip-10-194-9-233.us-west-2.compute.internal
              Creating org pedant_testorg_ip-10-194-9-233_31080
              Build step 'Invoke XShell command' marked build as failure
              Performing Post build task...
              Match found for : : True
            2540: |
              Recording fingerprints
              Recording test results
              ERROR: Publisher hudson.tasks.junit.JUnitResultArchiver aborted due to exception
              hudson.AbortException: No test report files were found. Configuration error?
                at hudson.tasks.junit.JUnitParser$ParseResultCallable.invoke(JUnitParser.java:116)
                at hudson.tasks.junit.JUnitParser$ParseResultCallable.invoke(JUnitParser.java:92)
                at hudson.FilePath$FileCallableWrapper.call(FilePath.java:2688)
                at hudson.remoting.UserRequest.perform(UserRequest.java:121)
                at hudson.remoting.UserRequest.perform(UserRequest.java:49)
                at hudson.remoting.Request$2.run(Request.java:325)
                at hudson.remoting.InterceptingExecutorService$1.call(InterceptingExecutorService.java:68)
                at java.util.concurrent.FutureTask.run(FutureTask.java:262)
                at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
                at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
                at java.lang.Thread.run(Thread.java:745)
                at ......remote call to chef-server-rhel-6-tester-8151c8(Native Method)
                at hudson.remoting.Channel.attachCallSiteStackTrace(Channel.java:1360)
                at hudson.remoting.UserResponse.retrieve(UserRequest.java:221)
                at hudson.remoting.Channel.call(Channel.java:753)
                at hudson.FilePath.act(FilePath.java:978)
                at hudson.FilePath.act(FilePath.java:967)
                at hudson.tasks.junit.JUnitParser.parseResult(JUnitParser.java:89)
                at hudson.tasks.junit.JUnitResultArchiver.parse(JUnitResultArchiver.java:121)
                at hudson.tasks.junit.JUnitResultArchiver.perform(JUnitResultArchiver.java:138)
                at hudson.tasks.BuildStepCompatibilityLayer.perform(BuildStepCompatibilityLayer.java:74)
                at hudson.tasks.BuildStepMonitor$1.perform(BuildStepMonitor.java:20)
                at hudson.model.AbstractBuild$AbstractBuildExecution.perform(AbstractBuild.java:761)
                at hudson.model.AbstractBuild$AbstractBuildExecution.performAllBuildSteps(AbstractBuild.java:721)
                at hudson.model.Build$BuildExecution.post2(Build.java:183)
                at hudson.model.AbstractBuild$AbstractBuildExecution.post(AbstractBuild.java:670)
                at hudson.model.Run.execute(Run.java:1766)
                at hudson.matrix.MatrixRun.run(MatrixRun.java:146)
                at hudson.model.ResourceController.execute(ResourceController.java:98)
                at hudson.model.Executor.run(Executor.java:374)
              [BFA] Scanning build for known causes...
              [BFA] No failure causes found
        steps:
          total: 8m18s
          before (setup time): 1m41s
          chef-client private-chef::default: 4m
          after (cleanup time): 2m36s
      el-7:
        result: FAILURE
        failure_category: unknown
        failure_cause: unknown
        url: http://wilson.ci.chef.co/job/chef-server-12-test/architecture=x86_64,platform=el-7,project=chef-server,role=tester/21/
        duration: 8m49s
        logs:
          consoleText:
            49749: |
              warning: file /opt/opscode/embedded/lib/ruby/gems/2.1.0/doc/nokogiri-1.6.3.1/ri/Nokogiri/XML/SyntaxError/int1-i.ri: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/ruby/gems/2.1.0/doc/nokogiri-1.6.3.1/ri/Nokogiri/XML/SyntaxError/file-i.ri: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/ruby/gems/2.1.0/doc/nokogiri-1.6.3.1/ri/Nokogiri/XML/SyntaxError/fatal%3f-i.ri: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/ruby/gems/2.1.0/doc/nokogiri-1.6.3.1/ri/Nokogiri/XML/SyntaxError/error%3f-i.ri: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/ruby/gems/2.1.0/doc/nokogiri-1.6.3.1/ri/Nokogiri/XML/SyntaxError/domain-i.ri: remove failed: No such file or directory
            58580: |
              warning: file /opt/opscode/embedded/lib/perl5/site_perl/5.18.1/x86_64-linux-thread-multi/auto/Test/File/.packlist: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/perl5/site_perl/5.18.1/x86_64-linux-thread-multi/auto/Test/File: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/perl5/site_perl/5.18.1/x86_64-linux-thread-multi/auto/Test/Fatal/.packlist: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/perl5/site_perl/5.18.1/x86_64-linux-thread-multi/auto/Test/Fatal: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/perl5/site_perl/5.18.1/x86_64-linux-thread-multi/auto/Test/Exception/.packlist: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/perl5/site_perl/5.18.1/x86_64-linux-thread-multi/auto/Test/Exception: remove failed: No such file or directory
            59523: |
              warning: file /opt/opscode/embedded/lib/perl5/site_perl/5.18.1/x86_64-linux-thread-multi/.meta/Test-File-1.44/MYMETA.json: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/perl5/site_perl/5.18.1/x86_64-linux-thread-multi/.meta/Test-File-1.44: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/perl5/site_perl/5.18.1/x86_64-linux-thread-multi/.meta/Test-Fatal-0.014/install.json: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/perl5/site_perl/5.18.1/x86_64-linux-thread-multi/.meta/Test-Fatal-0.014/MYMETA.json: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/perl5/site_perl/5.18.1/x86_64-linux-thread-multi/.meta/Test-Fatal-0.014: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/perl5/site_perl/5.18.1/x86_64-linux-thread-multi/.meta/Test-Exception-0.41/install.json: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/perl5/site_perl/5.18.1/x86_64-linux-thread-multi/.meta/Test-Exception-0.41/MYMETA.json: remove failed: No such file or directory
            59902: |
              warning: file /opt/opscode/embedded/lib/perl5/site_perl/5.18.1/Test/File.pm: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/perl5/site_perl/5.18.1/Test/File: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/perl5/site_perl/5.18.1/Test/Fatal.pm: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/perl5/site_perl/5.18.1/Test/Exception.pm: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/perl5/site_perl/5.18.1/Test/Dir/Base.pm: remove failed: No such file or directory
            63717: |
              warning: file /opt/opscode/embedded/lib/perl5/5.18.1/File/Basename.pm: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/perl5/5.18.1/File: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/perl5/5.18.1/Fatal.pm: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/perl5/5.18.1/ExtUtils/xsubpp: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/perl5/5.18.1/ExtUtils/typemap: remove failed: No such file or directory
            77724: |
              No entry for terminal type "unknown";
              using dumb terminal settings.
              /opt/opscode/embedded/service/oc-chef-pedant/lib/pedant/platform.rb:336:in `block in create_org': Bad error code 500 from create org: {"error":["internal service error"]} (RuntimeError)
                from /opt/opscode/embedded/service/oc-chef-pedant/lib/pedant/platform.rb:321:in `times'
                from /opt/opscode/embedded/service/oc-chef-pedant/lib/pedant/platform.rb:321:in `create_org'
            77737: |
              Configured URL: https://ip-10-194-10-226.us-west-2.compute.internal
              Creating org pedant_testorg_ip-10-194-10-226_4508
              Build step 'Invoke XShell command' marked build as failure
              Performing Post build task...
              Match found for : : True
            77758: |
              Recording fingerprints
              Recording test results
              ERROR: Publisher hudson.tasks.junit.JUnitResultArchiver aborted due to exception
              hudson.AbortException: No test report files were found. Configuration error?
                at hudson.tasks.junit.JUnitParser$ParseResultCallable.invoke(JUnitParser.java:116)
                at hudson.tasks.junit.JUnitParser$ParseResultCallable.invoke(JUnitParser.java:92)
                at hudson.FilePath$FileCallableWrapper.call(FilePath.java:2688)
                at hudson.remoting.UserRequest.perform(UserRequest.java:121)
                at hudson.remoting.UserRequest.perform(UserRequest.java:49)
                at hudson.remoting.Request$2.run(Request.java:325)
                at hudson.remoting.InterceptingExecutorService$1.call(InterceptingExecutorService.java:68)
                at java.util.concurrent.FutureTask.run(FutureTask.java:262)
                at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
                at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
                at java.lang.Thread.run(Thread.java:745)
                at ......remote call to chef-server-rhel-7-tester-d586ce(Native Method)
                at hudson.remoting.Channel.attachCallSiteStackTrace(Channel.java:1360)
                at hudson.remoting.UserResponse.retrieve(UserRequest.java:221)
                at hudson.remoting.Channel.call(Channel.java:753)
                at hudson.FilePath.act(FilePath.java:978)
                at hudson.FilePath.act(FilePath.java:967)
                at hudson.tasks.junit.JUnitParser.parseResult(JUnitParser.java:89)
                at hudson.tasks.junit.JUnitResultArchiver.parse(JUnitResultArchiver.java:121)
                at hudson.tasks.junit.JUnitResultArchiver.perform(JUnitResultArchiver.java:138)
                at hudson.tasks.BuildStepCompatibilityLayer.perform(BuildStepCompatibilityLayer.java:74)
                at hudson.tasks.BuildStepMonitor$1.perform(BuildStepMonitor.java:20)
                at hudson.model.AbstractBuild$AbstractBuildExecution.perform(AbstractBuild.java:761)
                at hudson.model.AbstractBuild$AbstractBuildExecution.performAllBuildSteps(AbstractBuild.java:721)
                at hudson.model.Build$BuildExecution.post2(Build.java:183)
                at hudson.model.AbstractBuild$AbstractBuildExecution.post(AbstractBuild.java:670)
                at hudson.model.Run.execute(Run.java:1766)
                at hudson.matrix.MatrixRun.run(MatrixRun.java:146)
                at hudson.model.ResourceController.execute(ResourceController.java:98)
                at hudson.model.Executor.run(Executor.java:374)
              [BFA] Scanning build for known causes...
              [BFA] No failure causes found
        steps:
          total: 8m49s
          before (setup time): 3m18s
          chef-client private-chef::default: 3m26s
          after (cleanup time): 2m4s
      ubuntu-10.04:
        result: FAILURE
        failure_category: code
        failure_cause: chef-client
        failed_in:
          step: chef-client
        url: http://wilson.ci.chef.co/job/chef-server-12-test/architecture=x86_64,platform=ubuntu-10.04,project=chef-server,role=tester/21/
        duration: 5m19s
        logs:
          consoleText:
            969: |
              [2015-12-29T14:33:52+00:00] INFO: Processing execute[restart_rabbitmq_log_service] action run (private-chef::rabbitmq line 19)
              [2015-12-29T14:33:52+00:00] INFO: execute[restart_rabbitmq_log_service] ran successfully
              [2015-12-29T14:33:52+00:00] ERROR: Running exception handlers
              [2015-12-29T14:33:52+00:00] ERROR: Exception handlers complete
              [2015-12-29T14:33:52+00:00] FATAL: Stacktrace dumped to /opt/opscode/embedded/cookbooks/cache/chef-stacktrace.out
              [2015-12-29T14:33:52+00:00] FATAL: Please provide the contents of the stacktrace.out file if you file a bug report
              [2015-12-29T14:33:52+00:00] FATAL: Mixlib::ShellOut::ShellCommandFailed: execute[/opt/opscode/embedded/bin/chpst -u opscode -U opscode /opt/opscode/embedded/bin/rabbitmqctl wait /var/opt/opscode/rabbitmq/db/rabbit@localhost.pid] (private-chef::rabbitmq line 110) had an error: Mixlib::ShellOut::ShellCommandFailed: Expected process to exit with [0], but received '2'
              ---- Begin output of /opt/opscode/embedded/bin/chpst -u opscode -U opscode /opt/opscode/embedded/bin/rabbitmqctl wait /var/opt/opscode/rabbitmq/db/rabbit@localhost.pid ----
              STDOUT: Waiting for rabbit@localhost ...
            992: |
              No entry for terminal type "unknown";
              using dumb terminal settings.
              /opt/opscode/embedded/service/oc-chef-pedant/lib/pedant/config.rb:36:in `from_argv': Configuration file '/var/opt/opscode/oc-chef-pedant/etc/pedant_config.rb' not found! (RuntimeError)
                from /opt/opscode/embedded/service/oc-chef-pedant/lib/pedant.rb:54:in `setup'
                from ./bin/oc-chef-pedant:10:in `<main>'
              Build step 'Invoke XShell command' marked build as failure
              Performing Post build task...
              Match found for : : True
            1016: |
              Recording fingerprints
              Recording test results
              ERROR: Publisher hudson.tasks.junit.JUnitResultArchiver aborted due to exception
              hudson.AbortException: No test report files were found. Configuration error?
                at hudson.tasks.junit.JUnitParser$ParseResultCallable.invoke(JUnitParser.java:116)
                at hudson.tasks.junit.JUnitParser$ParseResultCallable.invoke(JUnitParser.java:92)
                at hudson.FilePath$FileCallableWrapper.call(FilePath.java:2688)
                at hudson.remoting.UserRequest.perform(UserRequest.java:121)
                at hudson.remoting.UserRequest.perform(UserRequest.java:49)
                at hudson.remoting.Request$2.run(Request.java:325)
                at hudson.remoting.InterceptingExecutorService$1.call(InterceptingExecutorService.java:68)
                at java.util.concurrent.FutureTask.run(FutureTask.java:262)
                at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
                at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
                at java.lang.Thread.run(Thread.java:745)
                at ......remote call to chef-server-ubuntu-10.04-tester-f80d1f(Native Method)
                at hudson.remoting.Channel.attachCallSiteStackTrace(Channel.java:1360)
                at hudson.remoting.UserResponse.retrieve(UserRequest.java:221)
                at hudson.remoting.Channel.call(Channel.java:753)
                at hudson.FilePath.act(FilePath.java:978)
                at hudson.FilePath.act(FilePath.java:967)
                at hudson.tasks.junit.JUnitParser.parseResult(JUnitParser.java:89)
                at hudson.tasks.junit.JUnitResultArchiver.parse(JUnitResultArchiver.java:121)
                at hudson.tasks.junit.JUnitResultArchiver.perform(JUnitResultArchiver.java:138)
                at hudson.tasks.BuildStepCompatibilityLayer.perform(BuildStepCompatibilityLayer.java:74)
                at hudson.tasks.BuildStepMonitor$1.perform(BuildStepMonitor.java:20)
                at hudson.model.AbstractBuild$AbstractBuildExecution.perform(AbstractBuild.java:761)
                at hudson.model.AbstractBuild$AbstractBuildExecution.performAllBuildSteps(AbstractBuild.java:721)
                at hudson.model.Build$BuildExecution.post2(Build.java:183)
                at hudson.model.AbstractBuild$AbstractBuildExecution.post(AbstractBuild.java:670)
                at hudson.model.Run.execute(Run.java:1766)
                at hudson.matrix.MatrixRun.run(MatrixRun.java:146)
                at hudson.model.ResourceController.execute(ResourceController.java:98)
                at hudson.model.Executor.run(Executor.java:374)
              [BFA] Scanning build for known causes...
              [BFA] No failure causes found
        steps:
          total: 5m19s
          before (setup time): 2m23s
          after (cleanup time): 2m53s
      ubuntu-12.04:
        result: FAILURE
        failure_category: code
        failure_cause: chef-client
        failed_in:
          step: chef-client
        url: http://wilson.ci.chef.co/job/chef-server-12-test/architecture=x86_64,platform=ubuntu-12.04,project=chef-server,role=tester/21/
        duration: 7m23s
        logs:
          consoleText:
            967: |
              [2015-12-29T14:35:40+00:00] INFO: Processing execute[restart_rabbitmq_log_service] action run (private-chef::rabbitmq line 19)
              [2015-12-29T14:35:40+00:00] INFO: execute[restart_rabbitmq_log_service] ran successfully
              [2015-12-29T14:35:40+00:00] ERROR: Running exception handlers
              [2015-12-29T14:35:40+00:00] ERROR: Exception handlers complete
              [2015-12-29T14:35:40+00:00] FATAL: Stacktrace dumped to /opt/opscode/embedded/cookbooks/cache/chef-stacktrace.out
              [2015-12-29T14:35:40+00:00] FATAL: Please provide the contents of the stacktrace.out file if you file a bug report
              [2015-12-29T14:35:40+00:00] FATAL: Mixlib::ShellOut::ShellCommandFailed: execute[/opt/opscode/embedded/bin/chpst -u opscode -U opscode /opt/opscode/embedded/bin/rabbitmqctl wait /var/opt/opscode/rabbitmq/db/rabbit@localhost.pid] (private-chef::rabbitmq line 110) had an error: Mixlib::ShellOut::ShellCommandFailed: Expected process to exit with [0], but received '2'
              ---- Begin output of /opt/opscode/embedded/bin/chpst -u opscode -U opscode /opt/opscode/embedded/bin/rabbitmqctl wait /var/opt/opscode/rabbitmq/db/rabbit@localhost.pid ----
              STDOUT: Waiting for rabbit@localhost ...
            990: |
              No entry for terminal type "unknown";
              using dumb terminal settings.
              /opt/opscode/embedded/service/oc-chef-pedant/lib/pedant/config.rb:36:in `from_argv': Configuration file '/var/opt/opscode/oc-chef-pedant/etc/pedant_config.rb' not found! (RuntimeError)
                from /opt/opscode/embedded/service/oc-chef-pedant/lib/pedant.rb:54:in `setup'
                from ./bin/oc-chef-pedant:10:in `<main>'
              Build step 'Invoke XShell command' marked build as failure
              Performing Post build task...
              Match found for : : True
            1014: |
              Recording fingerprints
              Recording test results
              ERROR: Publisher hudson.tasks.junit.JUnitResultArchiver aborted due to exception
              hudson.AbortException: No test report files were found. Configuration error?
                at hudson.tasks.junit.JUnitParser$ParseResultCallable.invoke(JUnitParser.java:116)
                at hudson.tasks.junit.JUnitParser$ParseResultCallable.invoke(JUnitParser.java:92)
                at hudson.FilePath$FileCallableWrapper.call(FilePath.java:2688)
                at hudson.remoting.UserRequest.perform(UserRequest.java:121)
                at hudson.remoting.UserRequest.perform(UserRequest.java:49)
                at hudson.remoting.Request$2.run(Request.java:325)
                at hudson.remoting.InterceptingExecutorService$1.call(InterceptingExecutorService.java:68)
                at java.util.concurrent.FutureTask.run(FutureTask.java:262)
                at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
                at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
                at java.lang.Thread.run(Thread.java:745)
                at ......remote call to chef-server-ubuntu-12.04-tester-5b0c0e(Native Method)
                at hudson.remoting.Channel.attachCallSiteStackTrace(Channel.java:1360)
                at hudson.remoting.UserResponse.retrieve(UserRequest.java:221)
                at hudson.remoting.Channel.call(Channel.java:753)
                at hudson.FilePath.act(FilePath.java:978)
                at hudson.FilePath.act(FilePath.java:967)
                at hudson.tasks.junit.JUnitParser.parseResult(JUnitParser.java:89)
                at hudson.tasks.junit.JUnitResultArchiver.parse(JUnitResultArchiver.java:121)
                at hudson.tasks.junit.JUnitResultArchiver.perform(JUnitResultArchiver.java:138)
                at hudson.tasks.BuildStepCompatibilityLayer.perform(BuildStepCompatibilityLayer.java:74)
                at hudson.tasks.BuildStepMonitor$1.perform(BuildStepMonitor.java:20)
                at hudson.model.AbstractBuild$AbstractBuildExecution.perform(AbstractBuild.java:761)
                at hudson.model.AbstractBuild$AbstractBuildExecution.performAllBuildSteps(AbstractBuild.java:721)
                at hudson.model.Build$BuildExecution.post2(Build.java:183)
                at hudson.model.AbstractBuild$AbstractBuildExecution.post(AbstractBuild.java:670)
                at hudson.model.Run.execute(Run.java:1766)
                at hudson.matrix.MatrixRun.run(MatrixRun.java:146)
                at hudson.model.ResourceController.execute(ResourceController.java:98)
                at hudson.model.Executor.run(Executor.java:374)
              [BFA] Scanning build for known causes...
              [BFA] No failure causes found
        steps:
          total: 7m23s
          before (setup time): 3m34s
          after (cleanup time): 3m45s
      ubuntu-14.04:
        result: FAILURE
        failure_category: unknown
        failure_cause: unknown
        url: http://wilson.ci.chef.co/job/chef-server-12-test/architecture=x86_64,platform=ubuntu-14.04,project=chef-server,role=tester/21/
        duration: 7m48s
        logs:
          consoleText:
            2515: |
              No entry for terminal type "unknown";
              using dumb terminal settings.
              /opt/opscode/embedded/service/oc-chef-pedant/lib/pedant/platform.rb:336:in `block in create_org': Bad error code 500 from create org: {"error":["internal service error"]} (RuntimeError)
                from /opt/opscode/embedded/service/oc-chef-pedant/lib/pedant/platform.rb:321:in `times'
                from /opt/opscode/embedded/service/oc-chef-pedant/lib/pedant/platform.rb:321:in `create_org'
            2528: |
              Configured URL: https://ip-10-194-13-226.us-west-2.compute.internal
              Creating org pedant_testorg_ip-10-194-13-226_8364
              Build step 'Invoke XShell command' marked build as failure
              Performing Post build task...
              Match found for : : True
            2549: |
              Recording fingerprints
              Recording test results
              ERROR: Publisher hudson.tasks.junit.JUnitResultArchiver aborted due to exception
              hudson.AbortException: No test report files were found. Configuration error?
                at hudson.tasks.junit.JUnitParser$ParseResultCallable.invoke(JUnitParser.java:116)
                at hudson.tasks.junit.JUnitParser$ParseResultCallable.invoke(JUnitParser.java:92)
                at hudson.FilePath$FileCallableWrapper.call(FilePath.java:2688)
                at hudson.remoting.UserRequest.perform(UserRequest.java:121)
                at hudson.remoting.UserRequest.perform(UserRequest.java:49)
                at hudson.remoting.Request$2.run(Request.java:325)
                at hudson.remoting.InterceptingExecutorService$1.call(InterceptingExecutorService.java:68)
                at java.util.concurrent.FutureTask.run(FutureTask.java:262)
                at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
                at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
                at java.lang.Thread.run(Thread.java:745)
                at ......remote call to chef-server-ubuntu-14.04-tester-69fc04(Native Method)
                at hudson.remoting.Channel.attachCallSiteStackTrace(Channel.java:1360)
                at hudson.remoting.UserResponse.retrieve(UserRequest.java:221)
                at hudson.remoting.Channel.call(Channel.java:753)
                at hudson.FilePath.act(FilePath.java:978)
                at hudson.FilePath.act(FilePath.java:967)
                at hudson.tasks.junit.JUnitParser.parseResult(JUnitParser.java:89)
                at hudson.tasks.junit.JUnitResultArchiver.parse(JUnitResultArchiver.java:121)
                at hudson.tasks.junit.JUnitResultArchiver.perform(JUnitResultArchiver.java:138)
                at hudson.tasks.BuildStepCompatibilityLayer.perform(BuildStepCompatibilityLayer.java:74)
                at hudson.tasks.BuildStepMonitor$1.perform(BuildStepMonitor.java:20)
                at hudson.model.AbstractBuild$AbstractBuildExecution.perform(AbstractBuild.java:761)
                at hudson.model.AbstractBuild$AbstractBuildExecution.performAllBuildSteps(AbstractBuild.java:721)
                at hudson.model.Build$BuildExecution.post2(Build.java:183)
                at hudson.model.AbstractBuild$AbstractBuildExecution.post(AbstractBuild.java:670)
                at hudson.model.Run.execute(Run.java:1766)
                at hudson.matrix.MatrixRun.run(MatrixRun.java:146)
                at hudson.model.ResourceController.execute(ResourceController.java:98)
                at hudson.model.Executor.run(Executor.java:374)
              [BFA] Scanning build for known causes...
              [BFA] No failure causes found
        steps:
          total: 7m48s
          before (setup time): 2m25s
          chef-client private-chef::default: 3m17s
          after (cleanup time): 2m5s
  chef-server-12-build:
    result: SUCCESS
    url: http://wilson.ci.chef.co/job/chef-server-12-build/22/
    duration: 1h31m58s
    runs:
      el-5:
        result: SUCCESS
        url: http://wilson.ci.chef.co/job/chef-server-12-build/architecture=x86_64,platform=el-5,project=chef-server,role=builder/22/
        duration: 1h14m32s
      el-6:
        result: SUCCESS
        url: http://wilson.ci.chef.co/job/chef-server-12-build/architecture=x86_64,platform=el-6,project=chef-server,role=builder/22/
        duration: 1h30m57s
      el-7:
        result: SUCCESS
        url: http://wilson.ci.chef.co/job/chef-server-12-build/architecture=x86_64,platform=el-7,project=chef-server,role=builder/22/
        duration: 1h12m19s
      ubuntu-10.04:
        result: SUCCESS
        url: http://wilson.ci.chef.co/job/chef-server-12-build/architecture=x86_64,platform=ubuntu-10.04,project=chef-server,role=builder/22/
        duration: 1h1m34s
  chef-server-12-trigger-ad_hoc:
    result: SUCCESS
    url: http://wilson.ci.chef.co/job/chef-server-12-trigger-ad_hoc/12/
    duration: 6s
successful_logs_analyzed: false
